packages/orbit-components/src/SegmentedSwitch/__tests__/index.test.tsx

```diff
@@ -28,7 +27,7 @@ describe("SegmentedSwitch", () => {
     expect(screen.getByText(label)).toBeInTheDocument();
     expect(screen.getByTestId(dataTest)).toBeInTheDocument();
 
-    // $FlowFixMe
+    // @ts-expect-error TODO
     userEvent.tab(screen.getByLabelText("Male"));
```

Comment:
I don't know when this changed, but the `tab` function no longer receives the element to be tabbed. So we can remove the argument (and the TS annotation) and the test will still pass 🙂

Reply:
ah, right, thanks ;)